test/util/round.spec.ts

```
@@ -0,0 +1,112 @@
+  });
+
+  it('should handle very small numbers', () => {
+    expect(round(0.00001, 4)).toBe(0);
+    expect(round(0.00001, 5)).toBe(0.00001);
+    expect(round(0.00001, 6)).toBe(0.00001);
+    expect(round(1e-15, 6)).toBe(0);
+  });
+
+  it('should handle very large numbers', () => {
+    // eslint-disable-next-line no-loss-of-precision
+    expect(round(123456789.123456789, 2)).toBe(123456789.12);
```

Biome (2.1.2)

[error] This number literal will lose precision at runtime.

The value at runtime will be 123456789.12345679

(lint/correctness/noPrecisionLoss)